js/lib/configuration.js

```diff
@@ -15,6 +15,7 @@ elgg.get_site_url = function() {
  * @param {String} view    The full view name
  * @param {String} subview If the first arg is "css" or "js", the rest of the view name
  * @return {String} The site URL.
+ * @deprecated Use "require.toUrl(view)" inside your AMD modules instead.
```

Comment:
Are we sure toUrl() is sufficient? `foo/bar` might be a valid view for ajax HTML content, but I wonder if toUrl() will add `.js`...